src/trumbowyg.js

```diff
@@ -1433,6 +1440,24 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
 
                     t.syncCode();
                     t.semanticCode(false, true);
+                    try {
+                        var listId = window.getSelection().focusNode.parentNode;
+                        var arrayKey = param + 'Class';
```

Comment:
If we do like I said above, there is no need to build this key.

But, instead, we need to check if the tag has a class in the `tagClasses` object.

Reply:
is done